### PR TITLE
Change `__version__` location to `__init__` file

### LINKED ISSUE #16: Include `__version__` in `__init__.py` file
Following @jorgepiloto recommendation in https://github.com/pyansys/pyansys-tools-report/pull/13#pullrequestreview-1005273209 we should also make avaialbel the version in `__init__.py` to be in line with other pyansys libs. This will be considered as a new feature, no need to solve in released version.

### PR DESCRIPTION
**Pull request by RobPasMue**
Closes #16

### CODE REVIEW

File: tests/test_metadata.py [hunk 1]
```diff
@@ -1,7 +1,14 @@
-import ansys.tools.report as report
+import ansys.tools as ansys_tools
+import ansys.tools.report as ansys_tools_report
 
 
-def test_pkg_version():
+def test_pkg_version_method():
     """Test the correct functioning of the version() method."""
```
Comment (MaxJPRey) [inline tests/test_metadata.py:6]: @RobPasMue Do we still need this `version()` method as we now have the `__version__` attribute?

**Review by MaxJPRey — APPROVED** [review]

File: tests/test_metadata.py (hunk 1 above)
Comment (RobPasMue) [reply tests/test_metadata.py:6]: I mean, it's not "necessary" per se... But I'd like to give the option to users to choose their preferred option.

File: pyproject.toml [hunk 2]
```diff
@@ -41,7 +41,7 @@ doc = [
 home = "https://github.com/pyansys/pyansys-tools-report"
```
Comment (jorgepiloto) [inline pyproject.toml:41]: These links will be shown in the PyPI page of the package:

```suggestion
Home = "https://reportdocs.pyansys.com/"
Source = "https://github.com/pyansys/pyansys-tools-report"
Tracker = "https://github.com/ansys/pyansys-tools-report/issues"
```

**Review by jorgepiloto — APPROVED** [review]

File: pyproject.toml (hunk 2 above)
Comment (RobPasMue) [reply pyproject.toml]: Thanks! Adding to PR :)

**Pull request merged**